src/sfizz/Range.h

```diff
@@ -32,6 +32,12 @@ class Range {
     {
 
     }
+
+    constexpr Range(const Range<Type, !Checked>& other)
```

Comment:
Why the `!Checked` here?

Reply:
To allow `UncheckedRange` to be converted to `Range` and vice-versa